src/packages/@ncigdc/components/Charts/BarChart.js

```diff
@@ -18,6 +16,8 @@ import { withTooltip } from '@ncigdc/uikit/Tooltip';
 import withSize from '@ncigdc/utils/withSize';
 import './style.css';
 
+export const DEFAULT_X_AXIS_LENGTH = 8;
```

Comment:
thank you for naming this! can we add `LABEL` to the name? otherwise sounds like the axis range :)